### ISSUE TITLE
Can ASP.NET Core be hosted within ASP.NET using IIS Express in a development environment

### ISSUE THREAD

**Issue by philiphendry** [aspnet/Docs#8012]
I've managed to follow the article above and successfully configured a WebForms app to be available at http://localhost/WebApp and a ASP.NET Core app at http://localhost/WebApp/SubApp (both projects configured to startup). The reason for this nesting is so I can slowly migrate an old WebForms applications across to a SPA/MVC/WebApi architecture hosted in ASP.NET Core.

However, using IIS isn't an ideal development configuration and I'd prefer to continue using IIS Express. Is there an equivalent to the instructions above but that would work for IIS Express. I had tried changing the applicationUrl in launchSettings to simple be http://localhost:46288/WebApp/SubApp but that doesn't seem to work.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f5477f20-bdbf-ae0b-63b4-8bf6789f7a48
* Version Independent ID: fa9dc362-76a4-5d11-f457-b3d8550f9997
* Content: [Development-time IIS support in Visual Studio for ASP.NET Core](https://docs.microsoft.com/en-gb/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-2.1)
* Content Source: [aspnetcore/host-and-deploy/iis/development-time-iis-support.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/iis/development-time-iis-support.md)
* Product: **aspnet-core**
* GitHub Login: @shirhatti
* Microsoft Alias: **riande**

**Comment by shirhatti** [comment]
I don't believe we have first class support for this scenario, but I'm deferring to @BillHiebert 

@philiphendry I'd recommend creating a feature suggestion at https://developercommunity.visualstudio.com/content/idea/post.html for this ask.

**Comment by philiphendry** [comment]
I've made some progress with my original request and thought I'd mention what I've done. I created a Core web app project, set it's base path to /Core (`app.UsePathBase("/core")`) and then installed a proxy (`Install-Package Microsoft.AspNetCore.Proxy -Version 0.2.0`) and added this configuration:

```csharp
app.MapWhen(
                context => !context.Request.PathBase.Value.Equals(@"/core", StringComparison.OrdinalIgnoreCase),
                builder => builder.RunProxy(new ProxyOptions
                {
                    Scheme = "http",
                    Host = "localhost",
                    Port = "46287"
                }));
```
Now rather than using the original URL, http://localhost:46287, I send all requests to http://localhost/4855 instead. All urls are routed through to the old site except those starting with the `/core` base path.

I've only tried this in a test project where I'm experimenting with how to server Vue.JS from Core and WebForms but I think it's a solution that will work for me.
